docs/learn/learn-staking.md

```diff
@@ -6,110 +6,323 @@ description: An introduction on staking in Polkadot's NPoS consensus model.
 keywords: [staking, stake, nominate, nominating, NPoS]
 slug: ../learn-staking
 ---
+
 import RPC from "./../../components/RPC-Connection"
 
-{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} uses NPoS (Nominated Proof-of-Stake)
-as its [consensus](learn-consensus.md) mechanism. The system encourages 
-{{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} holders to participate
-as nominators. Nominators may back up to 
+Here you will lean about what is staking, why is important and how it works for
+{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }}.
+
+## Introduction
+
+Blockchain networks use [consensus](learn-consensus.md/#why-do-we-need-consensus) mechanisms to add
+blocks on the chain. Consensus is the process of agreeing on something, in this case the progression
+of the blockchain or how blocks are added to the chain. Consensus is split into two protocols:
+`block production`, i.e. the way multiple blocks candidates are produced, and `block finality`, i.e.
+the way only one block out of many candidates is selected and added to the chain. Because in public
+blockchains we have many participants who do not know each other (and probably never will), the
+reach of consensus happens in a secure and trustless way using Proof-of-Work (PoW) or Proof-of-Stake
+(PoS). In PoS networks like {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} the
+security of the network is dependent on the amount of capital locked on chain: the more the capital
+locked the lower the chance that someone will be able to successfully attack the network, as they
+would require a lot of own tokens or to collude different network participants. The process of
```

Comment:
Anyone who has significant stake in the network would not want to attack it, as it is not in their economic interests. Feel free to rephrase. The PoS network rewards good behaviour and punishes bad behaviour. Any rational actor staking on the network would want to get rewarded. It is not in their economic interests to orchestrate an attack on the network and risk losing their stake.

```suggestion
needs to incur heavy loss to orchestrate a successful attack. The process of
```

Reply:
Text added as requested